### ISSUE TITLE
Report: Explore the System information page

### ISSUE THREAD

**Issue by zwiastunsw** [joomla/accessibility#43]
## Review process
* **Scope of review**:
   * name of screen: System information
   * date of review: 19/07/2018
* **Reviewers**: Stefan Wajda
* **Evaluation environment**: 
   * Windows 10, 
   * Chrome 67.0.3396.99
* **Methods and tools**
   * **Tools**: aXe, Lighthouse, NVDA 
   * **Methods**: semi-automatic and automatic evaluation, manual evaluation, usability testing, keyboard only, assistive technology (screen reader)   
### Summary of results
The page is for information purposes only. **All informations are accessible**, but a number of issues have been identified that can cause problems. Repairing them can improve user experience.
### Strengths 
The data tables are of simple structure and are correctly marked.
#### 1. Admin menu
The menu module was not evaluated as [it is working on an improved version](https://codepen.io/littlesnippets/pen/2ee08f5688785b651887563ac630cc0c). However, please note that in the current menu there are several internal links named "collapse1", "collapse3", "collapse5", "collapse7", but no anchor exists with that name. However, there are anchors: "collapse2", "collapse4", "collapse6", "collapse8".
#### 2. This is not a form
The main content of this page is covered by the form tag. It does not contain any controls - no fields, no buttons. 
* **Recommendation**:  Consideration should be given to removing unnecessary form tag.
* **Success Criteria**: [4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#parsing), 
* **Suggested techniques**: 
   * [G192: Fully conforming to specifications](https://www.w3.org/TR/WCAG20-TECHS/G192.html)
   * [H88: Using HTML according to spec](https://www.w3.org/TR/WCAG20-TECHS/H88.html) 
#### 3. Redundant fieldset tags
The data tables are located inside the `fieldset` tag. The `fieldset` tag is intended for grouping the form controls, but the data table does not contain any form controls. The use of `fieldset` tags is incorrect and may be confusing. 
* **Recommendation**:  remove the redundant `fieldset` tags.
* **Success Criteria**: [4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#parsing), 
* **Suggested techniques**: 
   * [G192: Fully conforming to specifications](https://www.w3.org/TR/WCAG20-TECHS/G192.html)
   * [H88: Using HTML according to spec](https://www.w3.org/TR/WCAG20-TECHS/H88.html)  
#### 4. Headers instead of legend tags
Since the fieldset tags must be removed, the associated legend tags must also be removed. 
* **Recommendation**:  replace them with an h2 tags.
* **Success Criteria**: [4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#parsing), 
* **Suggested techniques**: 
   * [G192: Fully conforming to specifications](https://www.w3.org/TR/WCAG20-TECHS/G192.html)
   * [H88: Using HTML according to spec](https://www.w3.org/TR/WCAG20-TECHS/H88.html)  
#### 5. Redundant tfoot element
The tables contains an empty tfoot element. It is recommended to deleting them.  
#### 6. Toolbar - buttons marked as links. 
The two buttons in the toolbox are marked as links, although semantically they are buttons (buttons: Download as text, Downloads as JSON). The link is not a valid semantic tag in this case. It will misinform the blind user. A screen reader user expects to connect to a different page or location on a page, not an action. 
* **Recommendation**: change its to button (Use buttons to signal clickable actions, such as “download,” “sign up,” or “log out”). 
* **Success Criteria**: [4.1.1 Parsing (Level A)](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#parsing), 
* **Suggested techniques**: 
   * [G192: Fully conforming to specifications](https://www.w3.org/TR/WCAG20-TECHS/G192.html)
   * [H88: Using HTML according to spec](https://www.w3.org/TR/WCAG20-TECHS/H88.html) 
See: [Links vs. Buttons in Modern Web Applications](https://marcysutton.com/links-vs-buttons-in-modern-web-applications/)
#### 7. Toolbar - insufficient contrast
Button Help has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.04:1. 
* **Recommendation**: change background to #008297. 
* **Success Criteria**: [1.4.3 Contrast minimum](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-visual-audio-contrast-contrast). 
* **Suggested techniques**: [G18: Ensuring that a contrast ratio of at least 4.5:1](https://www.w3.org/TR/WCAG20-TECHS/G18.html)

**Comment by brianteeman** [comment]
When removing the fieldset the legend should be changed to be the table caption

**Comment by zwiastunsw** [comment]
Solved. See https://github.com/joomla/joomla-cms/pull/21724
https://github.com/joomla/joomla-cms/pull/23314
